sunpy/util/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+import numpy as np
+import sunpy.coordinates
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+
+def great_arc(a, b, center=SkyCoord(0*u.km, 0*u.km, 0*u.km, frame='heliocentric'), num=100):
```

Comment:
how often does one calculate great circles where the centre of the sphere is not the centre of the sun?

This might be better as a `center=None` kwarg so it doesn't look confusing in the docs.